app/lending/(1-concepts)/supply/page.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Supplying & Borrowing
+description: Supplying and borrowing in Chainflip Lending
+---
+
+## Supplying and Borrowing
+
+### Supplying Assets
+Chainflip Lending offers supply pools for supported assets. Liquidity suppliers deposit assets (most commonly USDC or USDT) into these pools, which are then made available to borrowers.
+
+All supply activity is per pool: users lend to a collective pool, not to individual borrowers. Interest and fees generated by borrowing activity are accrued to the pool and automatically increase the value of each supplier's share of the pool over time. No manual claiming is required, yields auto-compound.
+
+Suppliers use their Chainflip State Chain account to deposit assets directly from any supported EVM wallet. Deposited funds begin earning yield immediately based on pool utilisation and interest accrual.
+
+Supply-side risk is isolated per asset. Supplying USDC exposes the user only to USDC borrower risk and cannot be impacted by the behaviour of other markets.
+
+### Borrowing Assets
+Borrowers deposit collateral, typically BTC or ETH, but any supported collateral asset may be used, into their Chainflip State Chain account. This collateral can then be used to open one or more loan positions.
+
+Chainflip supports multi-collateral accounts, with loan health and liquidation behaviour determined using a per-account loan-to-value [LTV](/lending/collateralisation) ratio based on oracle prices.
+
+All collateral and debt in a borrower’s State Chain account share a single account-level LTV. If that LTV crosses a liquidation threshold, the protocol begins liquidating all open loan positions in the account, not just one of them.
+
+Auto-top-up is now an optional feature at the account level.
+
+Although the protocol supports it, auto-top-up will be disabled by default during the closed beta phase. This ensures testers do not unintentionally use their account balance to top up collateral during aggressive LTV conditions.
+
+If a borrower’s LTV approaches the soft liquidation threshold, the system automatically uses any free balance in their State Chain account to increase collateral and restore loan health.
+This reduces liquidation risk as long as sufficient free balance is available.
+
+Borrowers can also specify a **primary collateral asset**, which determines *which asset* will be used for top-up operations.
```

Comment:
We don't call it "primary collateral asset", but rather "topup collateral asset" or simply "topup asset". The user *must* specify it, otherwise the feature is considered disabled. Since it *is* the way to activate auto-topup, I would just talk about it from the start in the praragraph above.